workspaces/ui-v2/src/components/Panel.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC } from 'react';
+import { makeStyles } from '@material-ui/core';
+import { FontFamilyMono } from '<src>/styles';
+
+type PanelProps = {
+  header: React.ReactNode;
+};
+
+export const Panel: FC<PanelProps> = ({ children, header }) => {
+  const classes = useStyles();
+  return (
+    <div className={classes.wrapper}>
+      <div className={classes.header}>
+        <div>{header}</div>
+      </div>
+      <div className={classes.content}>{children}</div>
+    </div>
+  );
+};
+
+const useStyles = makeStyles((theme) => ({
+  wrapper: {
+    overflow: 'hidden',
+    width: '100%',
+  },
```

Comment:
These kinds of wrappers with full widths set and overflows to hidden generally worry me a bit, but I'm happy that more work needs to happen on the styling of these anyway :)